test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -448,6 +448,41 @@ public void PlattCalibratorOnnxConversionTest2()
             Done();
         }
 
+        [Fact]
+        public void TextNormalizingOnnxConversionTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-test.tsv");
+            var dataView = ML.Data.LoadFromTextFile(dataPath, new[] {
+                new TextLoader.Column("label", DataKind.Boolean, 0),
+                new TextLoader.Column("text", DataKind.String, 1)
+            }, hasHeader: true);
+            var pipeline = new TextNormalizingEstimator(mlContext, keepDiacritics: true, columns: new[] { ("NormText", "text") }).Append(
+                new TextNormalizingEstimator(mlContext, keepDiacritics: true, caseMode: TextNormalizingEstimator.CaseMode.Upper, columns: new[] { ("UpperText", "text") })).Append(
```

Comment:
Adding keepDiacritics only to the test code won't fix the issue. What happens if somebody creates a pipeline with a different option? I think this has to be handled in the SaveAsOnnx function for TextNormalizing

Reply:
I also qualified CanSaveOnnx(OnnxContext ctx) to return false for the other options. Is that not the expected behavior?

Reply:
Sorry, I missed that line. That sounds good.